CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Changelog
 
-## v1.5.3 (Unreleased)
+## v1.5.5 (2020-05-14)
```

Comment:
Hm, I see no release `v1.5.5` yet? :)

Reply:
it is almost ready :)

Reply:
released https://github.com/argoproj/argo-cd/releases/tag/v1.5.5